tests/smoke_tests/config.yaml

```diff
@@ -0,0 +1,25 @@
+# Parameters that describe server
```

Comment:
Since this is just going to be a smoke test, I'll suggest some config modifications here for it to run a bit faster.

`n_server_rounds` can be set to 2-3. This is the number of exchanges/aggregations done between server and clients. We want to test more than one exchange, but don't need to run 15.

`proximal_weight_patience` can be set to 1. If we're going to only be doing a few rounds, we want to make sure the `proximal_weight` is adjusted. Patience means that the algorithm waits a certain number of server rounds before making any changes.

Rather than using `local_epochs` we can use `local_steps` and set it to something small, maybe 5? Local epochs forces each client to run through all batches in the train set. If we use `local_steps` we can limit the number of train batches each client does.

I think you can safely remove the `reporting_config` section here. These are for W and B experiment tracking, which we don't need if we're just running a smoke test.